llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -173,17 +173,17 @@ cl::opt<module_split::IRSplitMode> SplitMode(
 cl::opt<bool> DoSymGen{"symbols", cl::desc("generate exported symbol files"),
                        cl::cat(PostLinkCat)};
 
-enum SpecConstMode { SC_USE_RT_VAL, SC_USE_DEFAULT_VAL };
+enum SpecConstLowerMode { SC_USE_NATIVE, SC_USE_EMULATION };
```

Comment:
not sure if we need this, but maybe some better renaming can be done, e.g.:
```suggestion
enum SpecConstLowerMode { SC_NATIVE, SC_EMULATION };
```
or
```suggestion
enum SpecConstLowerMode { SC_NATIVE_MODE, SC_EMULATION_MODE };
```